### DISCUSSION TITLE
release mode stuck

### DISCUSSION THREAD

**Discussion by ahmetbalaman** [google/model-viewer#4245] [Q&A]
Hi, I am trying this package on my phone with 
flutter run --release
mode but it isn't showing any model.

and another question is:
how can i open this model with ar mode?

**Comment by elalish** [comment]
I don't know much about flutter - can you give some details? Have you gotten it working properly on a simple website yet? That might be an easier place to start - then you'll know what is MV vs what is a strange flutter issue.
